geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -113,6 +113,8 @@
    */
   private volatile boolean _isStopped = true;
 
+  private volatile long wait_for_re_auth_start_time = -1;
```

Comment:
Let's do two things here. One `re` is too abbreviated. Spell it out. Also please switch to camel case even if some of the surrounding code uses the wrong case. (Camel case just means `waitForSomethingAuthStartTime`)

Reply:
Thanks! This PR is gonna be split into two PRs. I will incorporate your comment in the coming PR.